BitFaster.Caching/ConcurrentDictionarySize.cs

```diff
@@ -15,7 +15,7 @@ internal static int NextPrimeGreaterThan(int min)
                 }
             }
 
-            return 197;
+            return 7199369;
```

Comment:
We could simply return the input here, which is equivalent to the existing behavior. That way this is best efforts.